setup.py

```diff
@@ -22,13 +22,13 @@
     version += ".post" + str(int(time.time()))
 
 install_requires = [
-    "numpy>=1.14",
+    "numpy<1.19.0,>=1.16.0",
     "pandas>1.0.3,<2",
-    "scipy>=1.0.0,<2",
+    "scipy==1.4.1",
     "tqdm>=4.31.1,<5",
     "matplotlib>=2.2.2,<4",
     "scikit-learn>=0.22.1,<1",
-    "numba>=0.38.1,<1",
+    # "numba>=0.38.1,<1",
```

Comment:
mmm I think this would break the repo. Abir does numba conflicts with this code?

FYI @anargyri